### PR TITLE
Create Packer file system

### PR DESCRIPTION
**Pull request by Shazzberry**

### CODE REVIEW

File: packer/sources.pkr.hcl [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    # use the `name` field to name a build in the logs.
+    name          = var.build_name
+    ami_name      = var.ami_name
+    instance_type = var.instance_type
+    region        = var.aws_region
+
+    assume_role {
+        role_arn     = "arn:aws:sts::499504614116/AWSReservedSSO_PowerUserAccess_5ffa62a0c9832390"
+        session_name = "firefly_packer"
+    }
```
Comment (bbriggs) [inline packer/sources.pkr.hcl]: We aren't (yet) assuming a role

**Review by bbriggs — CHANGES_REQUESTED** [review]
If you change the name of the packer build file, you also need to update the GH action which is currently looking for `firefly.pkr.hcl`

**Comment by Shazzberry** [comment]
ready for another review i think

**Comment by bbriggs** [comment]
There is an existing conflict. Is this PR still valid?

**Pull request closed without merging**